tools/go-cli/README.md

```diff
@@ -1,15 +1,15 @@
-To build the wsk CLI run ant buildGoCLI from this directory.
+To build the OpenWhisk CLI run the following command from the OpenWhisk home directory:
 
-$ cd tools/go-cli
-$ ant buildGoCLI
+$ gradle :tools:go-cli:distDocker
 
-The build will produce the cli in ../bin/go-cli/$platform/$architecture/wsk
+Multiple binaries are produced in a Docker container during the build process. One of those binaries is copied from the
+Docker container to the local file system in the following directory: ../../bin/go-cli/wsk. This binary will be platform
+specific, it will only run on the operating system, and CPU architecture that matches the build machine's.
 
-The $platform is linux, mac, or windows.
+Currently the build process is only supported on Linux, and Mac operating systems running on an x86 CPU architecture.
```

Comment:
So I can't run the build process on my 64bit CPU Mac because my Mac is `x86_64`? :trollface: 
```
$ uname -a
Darwin carlosmacbookpro.raleigh.ibm.com 15.5.0 Darwin Kernel Version 15.5.0: Tue Apr 19 18:36:36 PDT 2016; root:xnu-3248.50.21~8/RELEASE_X86_64 x86_64
```